gvr-avatar/app/src/main/java/org/gearvrf/avatardemo/AvatarMain.java

```diff
@@ -27,45 +28,45 @@
 import org.joml.Quaternionf;
 import org.joml.Vector3f;
 
+import android.graphics.Color;
 import android.util.Log;
 import android.view.MotionEvent;
 
-public class AvatarMain extends GVRMain
-{
-//    private final String mModelPath = "DeepMotion/sahithi_character_skin.fbx";
-//    private final String mAnimationPath = "bodyturn.txt";
-//    private final String mModelPath = "Andromeda/Andromeda.dae";
-//    private final String[] mAnimationPaths =  { "Andromeda/HipHopDancing.dae", "Andromeda/Bellydancing.dae", "Andromeda/Boxing.dae" };
-//    private final String[] mAnimationPaths = { "DeepMotion/animation_baked.fbx" };
-//    private final String mModelPath = "Lily/female_outfitJ.fbx";
-//    private final String[] mAnimationPaths = { "Lily/Idle.fbx" };
-    private final String mModelPath = "astroboy/astro_boy.dae";
-    private final String[] mAnimationPaths =  { "astroboy/astro_boy.dae" };
+public class AvatarMain extends GVRMain {
```

Comment:
See gvr-x3d-demo and how we support setting the model to load via a system property (org.gearvrf.gvr-x3d-demo). Then all that commented out code can be removed and the models changed quicker.